Features CI

odiumuniverse opened the pull request:
1. Now Ci does not start twice at the same time on push and on PR
2. Tests are launched on several operating systems
3. Everything is launched on Oracle JDK

odiumuniverse committed:
Chage: Revised CI after Revoy

1. The launch is removed simultaneously immediately on PUSH and PR
2. CITEST is now launched on different operating systems
3. An extra Build team has been removed

odiumuniverse committed:
Change: Delete run on windows(feature on future)

odiumuniverse committed:
Change: MegaLinter run only on push

odiumuniverse committed:
Change: Download oracle jdk instead temurin

Ycyken commented on .github/workflows/CITest.yml:
```diff
@@ -0,0 +1,22 @@
+name: CITest
```
```suggestion
name: Build & Test
```

Ycyken committed:
Refactor: rename workflows files, rename gradle build workflow

Ycyken committed:
Refactor: rename job in gradle-build workflow

odiumuniverse committed:
Refactor: Delete .keep

The .keep folder was removed because it was only needed to send an empty directory
